client/src/components/BizCard_Map/BizCard_Map.jsx

```diff
@@ -0,0 +1,57 @@
+import React, {useState } from 'react';
+import { GoogleMap, Marker, InfoWindow } from '@react-google-maps/api';
+import flag from '../../images/hopper_flag.png'; 
+
+
+function BizCard_Map({props}){
+    //[we need update here]
+    //shop data is in the props. We can plot the spot
+    //by accessing props.lat and props.lng when  backend modify data. 
+    //for now, only Ebi-chan has location property
+    console.log("props",props)
+
+    //add lat, lng to props temporarily 
+    props.location = [{ lat:35.659871, lng:139.700662 }]
```

Comment:
i believe yusuke will be adding two columns called `lat` and `lng` so to access the data, it will be `biz.lat` and `biz.lng` ...but please confirm with him!